iib/workers/tasks/fbc_utils.py

```diff
@@ -17,6 +22,46 @@ def is_image_fbc(image):
     :return: True if image is FBC type, False otherwise (SQLite)
     :rtype: bool
     """
+    from iib.workers.tasks.utils import skopeo_inspect
+
     skopeo_output = skopeo_inspect(f'docker://{image}')
-    dc_image_label = 'operators.operatorframework.io.index.configs.v1'
-    return dc_image_label in skopeo_output['Labels']
+    fbc_image_label = 'operators.operatorframework.io.index.configs.v1'
+    return fbc_image_label in skopeo_output['Labels']
+
+
+def get_index_fbc_dir(from_index, base_dir):
+    """
+    Get file-based catalog directory from the specified index image and save it locally.
+
+    :param str from_index: index image to get file-based catalog directory from.
+    :param str base_dir: base directory to which the database file should be saved.
+    :return: path to the copied file-based catalog directory.
+    :rtype: str
+    :raises IIBError: if any podman command fails.
+    """
+    from iib.workers.tasks.build import _copy_files_from_image
+    from iib.workers.tasks.utils import get_image_label
+
+    fbc_dir_path = get_image_label(from_index, 'operators.operatorframework.io.index.configs.v1')
+    if not fbc_dir_path:
+        raise IIBError('Index image doesn\'t have the label specifying its catalog location.')
```

Comment:
Could you please log the label in the error message to make the error less ambiguous?

Reply:
Thanks!